API/ev3_timer.c

```diff
@@ -266,20 +266,29 @@ void TimerInit(unsigned long interval_multiplier)
 unsigned long long TimerGetUS()
 {
   struct timeval tv;
-  gettimeofday(&tv, 0);
-  return (((unsigned long long)tv.tv_sec) * 1000000) + tv.tv_usec;
+
+  if (gettimeofday(&tv, 0) == -1)

```

Comment:
While I don't mind more error checking, it seems unnecessary here. gettimeofday should always succeed when called like we do here.

Reply:
Replaced with a1f96dfe3f1a64569e832f7abc87b4d24439cd6c :+1: